x-pack/test/functional_with_es_ssl/apps/triggers_actions_ui/connectors/opsgenie.ts

```diff
@@ -140,6 +140,149 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
 
         expect(await (await testSubjects.find('executeActionButton')).isEnabled()).to.be(false);
       });
+
+      describe('test page', () => {
+        let connectorId = '';
+
+        before(async () => {
+          const connectorName = generateUniqueKey();
+          const createdAction = await createOpsgenieConnector(connectorName);
+          connectorId = createdAction.id;
+          objectRemover.add(createdAction.id, 'action', 'actions');
+        });
+
+        beforeEach(async () => {
+          await testSubjects.click(`edit${connectorId}`);
+          await testSubjects.click('testConnectorTab');
+        });
+
+        afterEach(async () => {
+          await actions.common.cancelConnectorForm();
+        });
+
+        it('should show the sub action selector when in test mode', async () => {
+          await testSubjects.existOrFail('opsgenie-subActionSelect');
+        });
+
+        it('should preserve the alias when switching between create and close alert actions', async () => {
+          await testSubjects.setValue('aliasInput', 'new alias');
+          await testSubjects.selectValue('opsgenie-subActionSelect', 'closeAlert');
+
+          expect(await testSubjects.getAttribute('opsgenie-subActionSelect', 'value')).to.be(
+            'closeAlert'
+          );
+          expect(await testSubjects.getAttribute('aliasInput', 'value')).to.be('new alias');
+        });
+
+        it('should not preserve the message when switching to close alert and back to create alert', async () => {
+          await testSubjects.setValue('messageInput', 'a message');
+          await testSubjects.selectValue('opsgenie-subActionSelect', 'closeAlert');
+
+          await testSubjects.missingOrFail('messageInput');
+          await retry.waitFor('message input to be displayed', async () => {
+            await testSubjects.selectValue('opsgenie-subActionSelect', 'createAlert');
```

Comment:
nit: Do we need to select the value on each retry?

Reply:
Yeah I was seeing flakiness with the form changing so I think reselecting the value seemed to help.